seleniumByTechnocredits/src/seleniumPractise/pallavi/Login_Password_Validation.java

```diff
@@ -0,0 +1,57 @@
+package seleniumPractise.pallavi;
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import util.Init;
+
+public class Login_Password_Validation 
+{
+
+	public static void main(String[] args) throws InterruptedException 
+	{
+		WebDriver driver = Init.initChromeDriver();
+		driver.get("http://automationbykrishna.com/#");
+		driver.manage().window().maximize();
+		Thread.sleep(3000);
+		driver.findElement(By.linkText("Registration")).click();
+		Thread.sleep(3000);
+		boolean flag = driver.findElement(By.xpath(".//*[text()='Login']")).isDisplayed();
+
+		if (flag == true) 
+		{
+			driver.findElement(By.xpath(".//input[@id='unameSignin' and @class ='form-control' ]")).sendKeys("Pallavi");
+			String password = "ab@123";
+			driver.findElement(By.xpath(".//input[@id='pwdSignin' and @class='form-control']")).sendKeys(password);
+			driver.findElement(By.xpath(".//button[@id='btnsubmitdetails']")).click();
+			if (password.length() >= 5 && password.length() <= 8) 
```

Comment:
Hi Pallavi,
Please consider below review comment:
1. Alert is also generating on password length less than 5. So the condition to math password length should only be less than or equal to 8.